src/nb_clean/__init__.py

```diff
@@ -274,6 +288,12 @@ def clean_notebook(
     if remove_empty_cells:
         notebook.cells = [cell for cell in notebook.cells if cell["source"]]
 
+    if remove_notebook_metadata:
+        notebook.metadata = {}
+    else:
+        with contextlib.suppress(KeyError):
+            del notebook["metadata"]["language_info"]["version"]
```

Comment:
The `else` branch here is a duplicate of the code on line 317, but that code is run under different conditions: it's run if `preserve_notebook_metadata` is False, whereas the code here is run if `remove_notebook_metadata` is False.